docker/imposter/agents/configuration/logcollector-socket.json

```diff
@@ -0,0 +1,4 @@
+{
+  "data": {},
```

Comment:
**question**: could we add some response? I guess the `data` object is empty because the purpose is the plugin doesn't fail on this request.

Reply:
I look for what data it would return and add it to that answer I got it from a freshly installed environment, so I had nothing.